generate_SNYK_vulnerabilities.py

```diff
@@ -0,0 +1,23 @@
+import json
+import os
+from pprint import pprint
+
+def BuildVulnerabilityMap():
+  CVE_Version = {} #CVE:{package_name:XXX, version:XXX}
```

Comment:
Do you mean only for the vulnerabilities that don't have a CVE ID or all of them? 

I don't think using them for all of them would be helpful because the search would be very expensive. Currently, since the mongo_handler passes in a vulnerability ID, I indexed the dictionary using the same. 

Can we change the mongo_handler (and the algorithm) to return an ID other than the CVE ID, if you do suggest changing all the IDs to SNYK ID?

Thanks!

Reply:
Yes, let's use the SNYK ID for all of them. I will change mongo_handler to do the same.

Reply:
There you go! 93c791d